php-frontend/src/test/java/org/sonar/php/PHPTreeModelTest.java

```diff
@@ -19,25 +19,14 @@
  */
 package org.sonar.php;
 
-import com.google.common.base.Charsets;
-import com.sonar.sslr.api.RecognitionException;
-import com.sonar.sslr.api.typed.ActionParser;
-import com.sonar.sslr.impl.ast.AstXmlPrinter;
-import org.sonar.php.parser.PHPLexicalGrammar;
 import org.sonar.php.parser.PHPParserBuilder;
-import org.sonar.php.tree.impl.PHPTree;
-import org.sonar.php.tree.impl.lexical.InternalSyntaxToken;
 import org.sonar.php.utils.SourceBuilder;
 import org.sonar.plugins.php.api.tree.Tree;
-import org.sonar.plugins.php.api.tree.Tree.Kind;
 import org.sonar.sslr.grammar.GrammarRuleKey;
-import org.sonar.sslr.internal.matchers.InputBuffer;
-import org.sonar.sslr.parser.ParseError;
-import org.sonar.sslr.parser.ParseErrorFormatter;
-
-import java.util.Iterator;
 
-import static org.fest.assertions.Assertions.assertThat;
+import com.google.common.base.Charsets;
+import com.sonar.sslr.api.RecognitionException;
+import com.sonar.sslr.api.typed.ActionParser;
 
 public class PHPTreeModelTest {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add some tests to this class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS2187)